.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Have you tried adding `singleQuote: true`?

Goal should be to make as few changes as possible while making it consistent, I see a ton of quote changes that don't really matter